components/description-list/description-list-wrapper.js

```diff
@@ -0,0 +1,100 @@
+import { bodyCompactStylize, labelStylize } from '../typography/styles.js';
+import { css, html, LitElement } from 'lit';
+import ResizeObserver from 'resize-observer-polyfill/dist/ResizeObserver.es.js';
+
+export const descriptionListStyles = [
+	labelStylize(css`dt`),
+	bodyCompactStylize(css`dd`),
+	css`
+		:host {
+			--d2l-dt-min-width: min-content;
+			--d2l-dt-max-width: 10rem;
+			--d2l-dd-min-width: 50%;
+		}
+		dl {
+			display: grid;
+			grid-auto-flow: row;
+			align-items: baseline;
+			grid-template-columns: minmax(var(--d2l-dt-min-width), auto) minmax(var(--d2l-dd-min-width), 1fr);
+			gap: 0.5rem 1.5rem;
+		}
+		dt {
+			max-width: var(--d2l-dt-max-width);
+		}
+		dd {
+			margin: 0;
+		}
+		d2l-dl-wrapper[stacked] dl {
+			display: block;
+		}
+		d2l-dl-wrapper[stacked] dt {
+			max-width: unset;
+		}
+		d2l-dl-wrapper[stacked] dd {
+			margin-bottom: 0.5rem;
+		}
+	`
+];
+
+/**
+ * Wraps a native <dl> element, providing styling and resize behavior.
+ * @slot - Content to wrap
+ */
+class DescriptionListWrapper extends LitElement {
+	static get properties() {
+		return {
+			stacked: { type: Boolean, reflect: true },
```

Comment:
It seems like we're not expecting consumers to set `stacked`, right? Like even if they do, the value will get clobbered by the the breakpoint handling code. So I'd suggest turning this into:

```suggestion
			_stacked: { state: true },
```

When then will result in the attribute no longer getting reflected to the host. That's both good (hosts "sprouting" attributes is kind of gross) and bad because then you can't target it using a CSS selector. To get around that, I'd suggest wrapping the `<slot>` in a `<div class="stacked">` (when it's stacked) and target that instead.

Reply:
Good point. I initially had the consumer set `stacked`, but when I added the breakpoint, it clobbers this.

I'm guessing the details of this implementation will depend on how we want "default" behavior to look (e.g. break at `__px`, don't break unless consumer specifies breakpoint, etc..)

Reply:
For sure yeah... I think `__px` is a reasonable place to start, and then we can iterator a bit later to see if there's a better approach. Nailing the default will be key too so that consumers almost never have to override.